### PR TITLE
Added cell count when viewing and editing the project

### PR DESCRIPTION
**Pull request by aaclan-ebi**
ebi-ait/dcp-ingest-central#344

### CODE REVIEW

**Comment by aaclan-ebi** [comment]
Screenshot 2021-06-09 at 12 56 52
Screenshot 2021-06-09 at 12 56 37

**Review by ke4 — DISMISSED** [review]
Looks fine by me 👍 💯

**Review by amnonkhen — APPROVED** [review]

**Review by jacobwindsor — APPROVED** [review]

File: client/src/app/project-form/project-ingest-schema.json [hunk 1]
```diff
@@ -14,17 +14,17 @@
...
       "type": "string",
       "format": "date-time",
-      "user_friendly": "Accession Time Frame"
+      "user_friendly": "Accession time frame"
     },
     "dataAccess": {
       "$schema": "http://json-schema.org/draft-07/schema#",
-      "description": "Data Access",
-      "title": "Data Access",
+      "description": "Data access",
```
Comment (amnonkhen) [inline client/src/app/project-form/project-ingest-schema.json:21]: I would argue that if we have a property called dataAccess, we can have the default title as "Data access" calculated programmatically, removing a lot of boilerplate code from this file.

**Review by amnonkhen — CHANGES_REQUESTED** [review]

File: client/src/app/project-form/project-ingest-schema.json (hunk 1 above)
Comment (ke4) [reply client/src/app/project-form/project-ingest-schema.json:21]: @amnonkhen I think you are right, but I would do it in another ticket. That change is a bit too much change in context of the current ticket. What do you think?

**Review by ke4 — APPROVED** [review]

File: client/src/app/project-form/project-ingest-schema.json (hunk 1 above)
Comment (aaclan-ebi) [reply client/src/app/project-form/project-ingest-schema.json:21]: yeah, i need to check where i could put that logic. I'll go with this for now!

Comment (aaclan-ebi) [reply client/src/app/project-form/project-ingest-schema.json:21]: @amnonkhen is it possible to convert the requested changes as a comment? it's not allowing me to merge unless I merge it as admin  🤔

Comment (aaclan-ebi) [reply client/src/app/project-form/project-ingest-schema.json:21]: `Merging can be performed automatically once the requested changes are addressed.`

**Pull request merged**